### PR TITLE
Gamedb: add EEtiminghack to 'MGS2' and change VU and EE rounding for 'Primal' and add missing entries 

### PR DESCRIPTION
**Pull request by Mrlinkwii**
### Description of Changes

add EE timinghack to 'MGS2' and change VU and EE rounding for 'Primal' and adding some missing entries 
### Rationale behind Changes

makes said games work better 
### Suggested Testing Steps

Check the CI /test said games

### CODE REVIEW

**Comment by gilderoylockhart01** [comment]
An other SPR while busy game. I wonder if in the future our dmac could be updated to handle this better.

**Comment by RedDevilus** [comment]
I don't see SPS on the trees anymore. Game sure needs a lot of love.

**Review by RedDevilus — APPROVED** [review]

**Comment by Mrlinkwii** [comment]
rebased and ready for merge :slightly_smiling_face:

File: bin/resources/GameIndex.yaml [hunk 1]
```diff
@@ -27827,6 +27871,9 @@ SLPM-66978:
   name: "Persona 4 [Konami-style Special Edition]"
   region: "NTSC-J"
   compat: 5
+SLPM-66980:
+  name: "Gin no Eclipse "
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:27875]: extra space

Comment (Mrlinkwii) [reply bin/resources/GameIndex.yaml]: fixed.

**Pull request merged**